tests/index.test.ts

```diff
@@ -204,6 +204,11 @@ describe('Result.Ok', () => {
     expect(okVal.unwrapOr(1)).toEqual(12)
   })
 
+  it('unwrapOrElse and return the Ok value', () => {
+    const okVal = ok(12)
+    expect(okVal.unwrapOrElse(() => 1)).toEqual(12)
+  })
```

Comment:
I think it's part of unwrapOrElse's contract to _not_ call the callback for an Ok value.

It could make sense to test that (using `jest.fn` and `expect(...).not.toHaveBeenCalled()`)